spec/requests/questionings_spec.rb

```diff
@@ -52,4 +52,71 @@
       end
     end
   end
+
+  describe "condition_form_data" do
+
+    let(:form) { create(:form, :published, question_types: %w(integer text integer text)) }
+    let(:qing) { form.questionings[2] }
+
+    context "without ref_qing_id" do
+      it "returns json with ref qing id options, no operator options, and no value options" do
+        expected_ref_qing_options = form.questionings.select{|q| q.id < qing.id }.map{ |q| { code: q.question.code, rank: q.full_dotted_rank, id: q.id } }
+        expected = {
+          id: nil,
+          ref_qing_id: nil,
+          op: nil,
+          value: nil,
+          form_id: form.id,
+          questioning_id: qing.id,
+          refable_qing_options: expected_ref_qing_options,
+          operator_options: [],
+          value_options: nil
+        }.to_json
+
+        get "/en/m/#{get_mission.compact_name}/questionings/condition-form",
+          { ref_qing_id: nil,
```

Comment:
I think styleguide calls for this brace to be at end of prev line, and then one level of indent for the attribs.